samples/Erlang/sample.app.src

```diff
@@ -0,0 +1,8 @@
+{application, sample,
```

Comment:
My bad. I selected the wrong grammar. It is indeed [highlighted properly](https://github-lightshow.herokuapp.com/?utf8=%E2%9C%93&scope=from-url&grammar_url=https%3A%2F%2Fgithub.com%2Ftextmate%2Ferlang.tmbundle%2Fblob%2F39c144ddb7a39bdb967c893a33fda27482beb770%2FSyntaxes%2FErlang.plist&grammar_text=&code_source=from-url&code_url=https%3A%2F%2Fgithub.com%2Fthwg%2Flinguist%2Fblob%2Fapp-src%2Fsamples%2FErlang%2Fsample.app.src&code=).